submitted_models/ctu_cras_norlab_x500_sensor_config_1/CMakeLists.txt

```diff
@@ -0,0 +1,55 @@
+cmake_minimum_required(VERSION 3.1.2)
+
+project(ctu_cras_norlab_x500_sensor_config_1)
+
+find_package(catkin REQUIRED COMPONENTS
+  roscpp
+  message_runtime
+  std_msgs
+  geometry_msgs
+)
+
+find_package(ignition-gazebo4 REQUIRED)
+find_package(ignition-transport9 REQUIRED)
+find_package(ignition-common3 REQUIRED)
+
+set(CMAKE_CXX_STANDARD 17)
+set(CMAKE_CXX_STANDARD_REQUIRED ON)
+set(CMAKE_EXPORT_COMPILE_COMMANDS ON)
+
+catkin_package(
+  CATKIN_DEPENDS message_runtime roscpp std_msgs geometry_msgs
+)
+
+include_directories(
+  include
+  ${catkin_INCLUDE_DIRS}
+)
+
+add_library(MRSMultirotorController SHARED
+  src/MRSMultirotorController.cpp
+  src/Common.cpp
+  src/SE3Controller.cpp
+)
+
+target_link_libraries(MRSMultirotorController
```

Comment:
I think the name of this library conflicts with `marble_qav500_sensor_config_1`. Compiling the entire workspace results in the following error:

```
CMake Error at subt/submitted_models/ctu_cras_norlab_x500_sensor_config_1/CMakeLists.txt:29 (add_library):
  add_library cannot create target "MRSMultirotorController" because another
  target with the same name already exists.  The existing target is a shared
  library created in source directory
  "/home/developer/subt_ws/src/subt/submitted_models/marble_qav500_sensor_config_1".
  See documentation for policy CMP0002 for more details.
```

Reply:
This seems strange, which tool are you building with?

Reply:
Sorry about that. We were developing the controller on the MARBLE_QAV500 UAV while creating our model in parallel. We left the controller in the MARBLE_QAV500 model by mistake. Should be fixed in https://github.com/osrf/subt/pull/811/commits/ce241b57566148722465148efd771eb1dfa96638.

I will address the rest of the issues soon.